JavaAssignments/src/assignment_12/Student.java

```diff
@@ -0,0 +1,53 @@
+package assignment_12;
+
+public class Student {
+    int id;
+    String name;
+    int age;
+    String gender;
+    String engDepartment;
+    int yoe;
+    double perDT;
+    public Student(int id,String name,int age,String gender,String engDepartment,int yoe,double perDT){
+        this.id = id;
+        this.name = name;
+        this.age = age;
+        this.gender = gender;
+        this.engDepartment=engDepartment;
+        this.yoe = yoe;
+        this.perDT = perDT;
+    }
+//    public String getName(){
+//        return name;
+//    }
+//    public int getId(){
+//        return id;
+//    }
+//    public String getGender(){
+//        return gender;
+//    }
+//    public int getAge(){
+//        return age;
+//    }
+//    public String getEngDepartment(){
```

Comment:
Any reason for commenting this part

Reply:
There were two methods to solve this problem, so I commented  one method.